walsh_a07b.js

```diff
@@ -0,0 +1,129 @@
+/* Tyler Walsh
+walsh_ao7b.js
+19WI_INFO_2124_WW
+Thoendel
+1/30/2019 */
+
+/* DO NOT MODIFY CODE BETWEEN THESE LINES */
+const standardInput = process.stdin;            
+standardInput.resume();
+standardInput.setEncoding('utf8');
+const badFoods = [];
+const groceryItems = [];
+clearScreen();
+console.log(getPrompt(groceryItems));
+/* DO NOT MODIFY CODE BETWEEN THESE LINES */
+
+//Step 1
+buildFoodsList(badFoods, "bread", "beer", "bacon", "butter", "beans"); //Invoking the buildFoodsList function to add these 5 items to the bad foods array 
+
+standardInput.on('data', function (text) {
+    //This line will remove line breaks 
+    //\n\r in window, \r in macOS
+    text = text.replace('\n', '').replace('\r', '');
+    //Step 4 Begin
+    if(text == "q" || text == "Q") { //If the user inputs a q in order to close the program

```

Comment:
you've got both checks! Good. You could use a switch here instead of the if/else - if you wanted.